graphrag/index/operations/embed_text/embed_text.py

```diff
@@ -142,24 +146,24 @@ async def _text_embed_with_vector_store(
     insert_batch_size: int = (
         vector_store_config.get("batch_size") or DEFAULT_EMBEDDING_BATCH_SIZE
     )
-    title_column: str = vector_store_config.get("title_column", "title")
-    id_column: str = vector_store_config.get("id_column", "id")
+
     overwrite: bool = vector_store_config.get("overwrite", True)
 
-    if column not in input.columns:
+    if embed_column not in input.columns:
+        msg = f"Column {embed_column} not found in input dataframe with columns {input.columns}"
+        raise ValueError(msg)
+    title = title_column or embed_column
```

Comment:
While trying to index, embed_text fails due to this change.
`title` is re-initialised in the while loop below leading to a key error. Using some other variable name will fix it.